include/ttmlir/Dialect/TTIR/IR/TTIROps.td

```diff
@@ -121,7 +121,7 @@ def TTIR_DeallocOp : TTIR_Op<"dealloc"> {
 //===----------------------------------------------------------------------===//
 
 class TTIR_ElementwiseOp<string mnemonic, list<Trait> traits = []> :
-    TTIR_DPSOp<mnemonic, !listconcat(traits, [Elementwise, AttrSizedOperandSegments, TTIR_ElementwiseOpInterface])> {
```

Comment:
Please add comment explaining implicit broadcast behaviour for these type of ops. If possible double check memory usage behaviour of OP which consumes and broadcasts one of the operands, ie is memory usage derived from pre or post broadcast shape.

Reply:
Added comment about the broadcast. I spent some time digging through tt-metal to find out memory consumption in this case, but I couldn't find definite answer. I feel it shouldn't affect L1 memory as broadcast is only done on input operands which already live in DRAM or L1 (output of a prev. op). One more thing, bcast kernel is located in ttnn/deprecated dir, which is kinda suspicious. I propose to allow this broadcast for now, and see if things break down once we start onboarding more complex models.